### ISSUE TITLE
Show points earned after finishing each scenario

### ISSUE THREAD

**Issue by alma-castant** [systers/powerup-android#16]
After a scenario is completed, the user should be shown all the points she or he has earned during that specific scenario

**Comment by chhavip** [comment]
@alma-castant , the points have to be displayed on a separate activity or a dialog that opens after each scenario finishes?

**Comment by alma-castant** [comment]
It should be a separate activity after a single scenario is finished

**Comment by chhavip** [comment]
I have designed the screen and added functional code, the page looks like this: (points will be displayed accordingly)

![display points](https://cloud.githubusercontent.com/assets/7212638/12214774/913fc1f8-b6c7-11e5-9f31-2efb7970d65d.png)

On clicking Map, the user is directed back to MapActivity while on clicking Next Scenario, the next scenario is loaded right after finish of this activity. If the changes sound okay, should I submit a PR?

**Comment by alma-castant** [comment]
Looks good. I would maybe rephrase to "Brilliant! Scenario completed. Here are the points you earned:" Feel free to submit a PR. There is currently a bug in how scenarios are handled, so if that becomes an issue when calculating the points, feel free to leave this for later

**Comment by chhavip** [comment]
@alma-castant, I displayed points for a few scenarios and they seemed to be working fine, I will submit a PR and you can review.

**Comment by chhavip** [comment]
Fixed.
